.travis.yml

```diff
@@ -11,8 +12,6 @@ matrix:
       env:
         DEPLOYMENT_ENV="true"
         BADGE=linux
-      services:
```

Comment:
If we are only running docker tests on linux for now, I assume we should revert to docker only being a service undex linux ?